binder/plugins/models/html_field.py

```diff
@@ -0,0 +1,80 @@
+from django.db.models import TextField
+from html.parser import HTMLParser
+from django.core import exceptions
+
+
+def link_validator(tag, attribute_name, attribute_value):
+	if not attribute_value.startswith('http://') and not attribute_value.startswith('https://'):
```

Comment:
Do we not want to allow linking to for instance `ftp://` (potential security risk) or if possible `sftp://`? or for example mailing via a link with a `href="mailto:foo@bar.com"`

Reply:
But would adding another special validator then still not cause this link validator to throw an exception? thus cancelling the transaction and disabling saving of the field? or would you expect overwriting of the entire `special_validator` dict?

because the disadvantage of that would be that you lose any other default validators, as well as future validators should those be updated in binder in the future.

I would personally be in favor of adding `mailto` to be honest, since i would expect it to be quite a widely used `a` `href`.